src/modals/CenteredModal.tsx

```diff
@@ -191,7 +191,7 @@ const BackButton = styled(Button)`
 export const ModalContent = styled.div`
   display: flex;
   flex-direction: column;
-  padding: var(--spacing-4);
+  padding: var(--spacing-4) var(--spacing-4) 0 var(--spacing-4);
```

Comment:
Why this change? Now the Action buttons are really close to the modal bottom and it's less balanced IMO.

Reply:
I've just seen the comment [here](https://github.com/alephium/desktop-wallet/commit/978aa5b71ccf3666beb7174c88dd4358b1808317). Yeah unfortunately I feel like it lacks space compared to the rest of the UI.

Reply:
I understand. If you don't mind tackling this while I work on the WC issue, I'd appreciate it a lot! I'll focus my attention there now.